### PR TITLE
bug 1971068: Updating AWS instance types

### PR DESCRIPTION
**Pull request by kalexand-rh**
As a follow-up to docs bug BZ1878374, it was pointed out that two
of the instance types listed in the Cloud Formation templates do
not exist. This change corrects the instance types to those
proposed on the corresponding docs PR.

See https://github.com/openshift/openshift-docs/pull/33358

@cuppett 

This change also needs to apply to 4.7 and 4.8.

https://bugzilla.redhat.com/show_bug.cgi?id=1971068

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@patrickdillon, @jstuever, will one of you PTAL and help me get this into `release-4.7` and `release-4.8` as well?

File: upi/aws/cloudformation/05_cluster_master_nodes.yaml [hunk 1]
```diff
@@ -55,14 +55,14 @@ Parameters:
     - "m4.xlarge"
     - "m4.2xlarge"
     - "m4.4xlarge"
-    - "m4.8xlarge"
+    - "m4.10xlarge"
```
Comment (patrickdillon) [inline upi/aws/cloudformation/05_cluster_master_nodes.yaml:58]: `m4.10xlarge` is already in the list

File: upi/aws/cloudformation/06_cluster_worker_node.yaml [hunk 2]
```diff
@@ -37,15 +37,15 @@ Parameters:
     - "m4.xlarge"
     - "m4.2xlarge"
     - "m4.4xlarge"
-    - "m4.8xlarge"
+    - "m4.10xlarge"
```
Comment (patrickdillon) [inline upi/aws/cloudformation/06_cluster_worker_node.yaml:40]: `m4.10xlarge` is already in the list

Comment (kalexand-rh) [reply upi/aws/cloudformation/06_cluster_worker_node.yaml:40]: @patrickdillon, thank you! Will you PTAL?

**Comment by cuppett** [comment]
+1 on getting these to match the actual steppings.

**Pull request merged**